libc/src/setjmp/arm/longjmp.cpp

```diff
@@ -0,0 +1,30 @@
+
+//===-- Implementation of longjmp -----------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/setjmp/longjmp.h"
+#include "src/__support/common.h"
+#include "src/__support/macros/properties/architectures.h"
+
+#if !defined(LIBC_TARGET_ARCH_IS_ARM)
+#error "Invalid file include"
+#endif
```

Comment:
These warnings (pattern copied from other setjmp impl's in tree) are dumb. You'll get an obvious compiler diagnostic without it pretty quick! I'm going to remove them.

Reply:
77384336fb4e